### ISSUE TITLE
Introduce maximum amount of submissions per devroom and in total

### ISSUE THREAD

**Issue by RichiH** [FOSDEM/pretalx-integration#37]
Some people submit a dozen proposals or more. This is unfair to other submitters, increases load on reviewers, and does not instill confidence that the submitter thought about the topics they can best speak on.

**Comment by kergon** [comment]
On Tue, Dec 19, 2023 at 06:10:12AM -0800, Richard Hartmann wrote:
>    Some people submit a dozen proposals or more. This is unfair to other
>    submitters, increases load on reviewers, and does not instill confidence
>    that the submitter thought about the topics they can best speak on.
 
That's probably an isolated problem that doesn't need a tool change and can 
be handled by screening the track's submissions before review.

Perhaps formalise it by adding to your CfP next year that only the first 3
submissions from any one individual will be considered?

Other tracks might be happy to have lots of submissions to choose from.

Alasdair

**Comment by johanvdw** [comment]
I would not enforce this in the tool, because it adds complexity. Eg what to do if a talk has multiple speakers.

Do you realize you have submitted 4 talks as well?

**Comment by RichiH** [comment]
Yes, I do, and will submit a fifth one (short intro to devroom); opening/closing/etc are a bit of a special case, though
